addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -501,6 +501,30 @@ def test_message_route_alias_owner_author_notify(self):
 
             messages = test_record.message_ids
 
+    @mute_logger('odoo.addons.mail.models.mail_thread', 'odoo.models')
+    def test_message_route_bounce_reset(self):
```

Comment:
Can probably update ``test_message_process_bounce_records_channel`` to avoid adding yet another test

Reply:
> Can probably update `test_message_process_bounce_records_channel` to avoid adding yet another test

Thanks done :) 
I allowed myself to add few comments aswell